lib/IDE/ReconstructType.cpp

```diff
@@ -22,6 +22,10 @@
 #include <cstdio>
 #include <mutex> // std::once
 
+#if defined(__FreeBSD__)
+#include <stdarg.h>
+#endif
```

Comment:
I think we should drop the `#if` since currently we are relying on transitive includes, which are unstable.